Tests/XCUITests/PrivateBrowsingTest.swift

```diff
@@ -67,14 +67,14 @@ class PrivateBrowsingTest: BaseTestCase {
         navigator.nowAt(NewTabScreen)
         waitForTabsButton()
         navigator.goto(TabTray)
-        waitForExistence(app.cells.staticTexts[url1And3Label])
+        waitForExistence(app.otherElements["Tabs Tray"].cells.staticTexts[url1And3Label])
         let numPrivTabs = app.otherElements["Tabs Tray"].cells.count
         XCTAssertEqual(numPrivTabs, 1, "The number of private tabs is not correct")
         // Go back to regular mode and check the total number of tabs
         navigator.toggleOff(userState.isPrivate, withAction: Action.ToggleRegularMode)
 
-        waitForExistence(app.cells.staticTexts[url2Label])
-        waitForNoExistence(app.cells.staticTexts[url1And3Label])
+        waitForExistence(app.otherElements["Tabs Tray"].cells.staticTexts[url2Label])
+        waitForNoExistence(app.otherElements["Tabs Tray"].cells.staticTexts[url1And3Label])
         let numRegularTabs = app.otherElements["Tabs Tray"].cells.count
```

Comment:
Sometimes I was not getting the correct number here. I would suggest to add collectionViews to get the count:
app.otherElements["Tabs Tray"].collectionViews.cells.count